src/org/usfirst/frc/team4915/steamworks/OI.java

```diff
@@ -66,24 +68,23 @@ public OI(Robot robot)
         }
     }
 
+    public Command getAutoCommand()
+    {
+        return null;
```

Comment:
I don't think the error I was observing earlier was actually due to this, patched to return null now.